fdbclient/ServerKnobs.cpp

```diff
@@ -864,16 +864,18 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( LATENCY_METRICS_LOGGING_INTERVAL,                     60.0 );
 
 	// Cluster recovery
-	init ( CLUSTER_RECOVERY_EVENT_NAME_PREFIX,               "Master");
+	init ( CLUSTER_RECOVERY_EVENT_NAME_PREFIX,              "Master" );
 
-    // encrypt key proxy
-    init( ENABLE_ENCRYPTION,                                   false );
-    init( ENCRYPTION_MODE,                              "AES-256-CTR");
-    init( SIM_KMS_MAX_KEYS,                                      4096);
-    init( ENCRYPT_PROXY_MAX_DBG_TRACE_LENGTH,                  100000);
+	// encrypt key proxy
+	init( ENABLE_ENCRYPTION,                                   false ); if ( randomize && BUGGIFY ) { ENABLE_ENCRYPTION = deterministicRandom()->coinflip(); }
+	init( ENCRYPTION_MODE,                             "AES-256-CTR" );
+	init( SIM_KMS_MAX_KEYS,                                     4096 );
+	init( ENCRYPT_PROXY_MAX_DBG_TRACE_LENGTH,                 100000 );
+	init( ENABLE_TLOG_ENCRYPTION,                  ENABLE_ENCRYPTION ); if ( randomize && BUGGIFY) { ENABLE_TLOG_ENCRYPTION = (ENABLE_ENCRYPTION && !PROXY_USE_RESOLVER_PRIVATE_MUTATIONS && deterministicRandom()->coinflip()); }
+	init( ENCRYPT_KEY_REFRESH_INTERVAL,                         10.0 );
```

Comment:
a nit: Flow/Knobs.h define 

```
	// Encryption
	int64_t ENCRYPT_CIPHER_KEY_CACHE_TTL;
	int64_t ENCRYPT_KEY_REFRESH_INTERVAL;
```
Do they have similar usage? If yes, we should consolidate them (as a separate PR is fine as well)

Reply:
Actually I'm not using this knob. Let me clean it up.